docs/apache-airflow/authoring-and-scheduling/datasets.rst

```diff
@@ -382,6 +382,64 @@ For scenarios requiring more intricate conditions, such as triggering a DAG when
     ):
         ...
 
+
+Dynamic data events emitting and dataset creation through DatasetAlias
+-----------------------------------------------------------------------
+A dataset alias can be used to emit dataset events of datasets with association to the aliases. Downstreams can depend on resolved dataset. This feature allows you to define complex dependencies for DAG executions based on dataset updates.
+
+How to use DatasetAlias
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
How to use DatasetAlias
~~~~~~~~~~~~~~~~~~~~~~~
```

Hmm I thought there’s a lint rule about the underline should be of the same length of the text, but I guess we removed it at some point. Nit, not important.

Reply:
There's one for too short, but probably not too long